src/parse-mockdb.js

```diff
@@ -511,7 +511,7 @@ function queryMatchesAfterIncluding(matches, includeClause) {
 function includePaths(object, pathsRemaining) {
   debugPrint('INCLUDE', {object, pathsRemaining})
   const path = pathsRemaining.shift();
-  const target = object[path];
+  const target = !object ? null : object[path];
```

Comment:
actually, I'm sorry i didn't realize this was just to protect object[path], lets do `object && object[path]`